exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtilsTest.java

```diff
@@ -185,6 +185,26 @@ public void createMetric() {
                 .build());
   }
 
+  @Test
+  public void createMetric_WithExternalMetricDomain() {
+    View view =
+        View.create(
+            Name.create(VIEW_NAME),
+            VIEW_DESCRIPTION,
+            MEASURE_DOUBLE,
+            DISTRIBUTION,
+            Arrays.asList(KEY),
+            CUMULATIVE);
+    String prometheusDomain = "external.googleapis.com/prometheus/";
```

Comment:
/cc @jkohen 

Does this match what you suggested in https://github.com/census-instrumentation/opencensus-go/issues/836?

Reply:
Yes, thank you.